### DISCUSSION TITLE
How to access drag & drop TOC in Beta?

### DISCUSSION THREAD

**Discussion by kenanmike** [Zettlr/Zettlr#2412] [General]
One of my main uses for Zettlr is the drag and drop TOC but in the new Beta it's no longer there on the top bar. Anyway to recover it?

**Comment by nathanlesage** [comment]
Not been re-implemented yet – I have this on my radar, and I'll modify the sidebar table of contents so rearranging documents is possible again. But, as I said on Twitter, this is surprisingly difficult, so it might take a while until I get to it. But it will come back.

**Comment by kenanmike** [comment]
Thanks very much Hendrik!
Sounds great. Having one of those (d&d TOC in sidebar or d&d TOC in the top bar) is far more important to me than having both. And I think the sidebar is preferable to the top bar because it can stay open and in a way that the doc is still fully visible.
